RELEASE.md

```diff
@@ -0,0 +1,28 @@
+## Quality gates
+
+| Risk level                    | Edge                            | Beta                         | Candidate                                                                                    | Stable                                                                                                                                            |
+|-------------------------------|---------------------------------|------------------------------|----------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| Meaning                       | Bleeding edge developer version | Most new features stabilized | Feature-ready, currently in testing                                                          | Well-tested, production-ready                                                                                                                     |
+| Preconditions (quality gates) | Code review, CI                 |                              | Load tests                                                                                   |                                                                                                                                                   |
+| Timing                        | Every merge to main             |                              | Charm reaches a state of feature completion with respect to the next planned stable release. | In consultation with product manager and engineering manager when the candidate revision has been well tested and is deemed ready for production. |
+| Release process               | Automatic (on merge)            | Manual                       | Manual                                                                                       | Manual                                                                                                                                            |
+
+
+## Publishing
+```shell
+./render_bundle.py bundle.yaml --channel=edge
```

Comment:
edge is the default, so no need to supply it. also, `tox -e render` is a thing now :)

Reply:
Seems like we both did this then 😅 https://github.com/canonical/cos-lite-bundle/pull/36/commits/d761a09216021dbb91cb1f4924b5ea4af956968b